### ISSUE TITLE
Weird behavior when specifying "on" in `identifier_name` excluded list

### ISSUE THREAD

**Issue by nvh** [realm/SwiftLint#1424]
```
identifier_name:
  excluded:
    - to
    - id
    - ok
    - on
```
This stopped working since 0.18.1. Swiftlint now acts as if the excluded list doesn't exist. However, when I remove `on` from this list, it works as expected (but still giving a warning about the use of `on` in my codebase of course).

 I think this is the result of the switch in YAML parsing, because "on" is a reserved word in yaml used to specify a boolean. I've tried putting it in quotes around `"on"` but that didn't help.

**Comment by norio-nomura** [comment]
> because "on" is a reserved word in yaml used to specify a boolean.

You are right.
Could you try using `- !str on`? That instructs the parser to explicitly parse it as a string.

We may have to customize loading of configuration to avoid this.

**Comment by zac** [comment]
Same issue for me with "no". Prefixing with `- !str` works for me.
